src/main/java/com/zendesk/maxwell/schema/ddl/MysqlParserListener.java

```diff
@@ -283,18 +283,18 @@ else if ( tokenText.equals(")") )
 	/* we enter this code twice.  the first time, we gobble up parens.  The
 		   second time, we just have the __MAXWELL__ token, and we can continue.
 		 */
-//	@Override
-//	public void enterSkip_parens(Skip_parensContext ctx) {
-//		if ( ctx.MAXWELL_ELIDED_PARSE_ISSUE() == null )
-//			throw new ReparseSQLException(spliceParens(ctx.getStart().getTokenIndex(), 0));
-//
-//	}
-//
-//	@Override
-//	public void enterSkip_parens_inside_partition_definitions(Skip_parens_inside_partition_definitionsContext ctx) {
-//		if ( ctx.MAXWELL_ELIDED_PARSE_ISSUE() == null )
-//			throw new ReparseSQLException(spliceParens(ctx.getStart().getTokenIndex(), 1));
-//	}
+	@Override
+	public void enterSkip_parens(Skip_parensContext ctx) {
+		if ( ctx.MAXWELL_ELIDED_PARSE_ISSUE() == null )
+			throw new ReparseSQLException(spliceParens(ctx.getStart().getTokenIndex(), 0));
+
+	}
+
+	@Override
+	public void enterSkip_parens_inside_partition_definitions(Skip_parens_inside_partition_definitionsContext ctx) {
+		if ( ctx.MAXWELL_ELIDED_PARSE_ISSUE() == null )
+			throw new ReparseSQLException(spliceParens(ctx.getStart().getTokenIndex(), 1));
+	}
 
```

Comment:
@osheroff make via intellij. I found that it doesn't actually pickup changes in the filters unless I use make on the cmdline so it's gotta be an IDE config.